R/learner_survival_surv_parametric.R

```diff
@@ -284,4 +299,57 @@ LearnerSurvParametric = R6Class("LearnerSurvParametric",
   list(lp = as.numeric(lp), distr = distr)
 }
 
+
```

Comment:
I cannot review this, I guess raphael did this part?

Reply:
Yes, I tested it and found some very edge cases that needed fixing and fixed the test code

Reply:
great! :)